assets/styles.css

```diff
@@ -0,0 +1,930 @@
+button {
+  border: none;
+  font-weight: bold;
```

Comment:
Replace with bootstrap classes

Reply:
Adding `btn` bootstrap class to  all button elements